src/jmespath.net.parser/jmespath.net.parser.csproj

```diff
@@ -20,32 +19,34 @@
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
   </PropertyGroup>
   <ItemGroup>
+    <YaccFile Include="JmesPathParser.y">
+      <OutputFile>JmesPathParser.g.cs</OutputFile>
+    </YaccFile>
+    <LexFile Include="JmesPathScanner.lex">
+      <OutputFile>JmesPathScanner.g.cs</OutputFile>
+    </LexFile>
     <Compile Include="JmesPathScanner.g.cs" Condition="!Exists( 'JmesPathScanner.g.cs' )" />
     <Compile Include="JmesPathParser.g.cs" Condition="!Exists( 'JmesPathParser.g.cs' )" />
+	<Compile Include="GplexBuffers.cs" Condition="!Exists( 'GplexBuffers.cs' )" />
+	<Compile Include="ShiftReduceParserCode.cs" Condition="!Exists( 'ShiftReduceParserCode.cs' )" />
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\jmespath.net.interop\jmespath.net.interop.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Springcomp.GPLEX.Runtime" Version="1.2.4" />
-    <PackageReference Include="Springcomp.GPPG.Runtime" Version="1.2.4" />
-  </ItemGroup>
-
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
-    <PackageReference Include="System.Reflection.TypeExtensions" Version="4.7.0" />
+	  <!-- Use GplexBuffers and ShiftReduceParserCode from the net13 folder instead, which are modified for use of netstandard 1.3 -->
+	  <Compile Remove="ShiftReduceParserCode.cs" />
+	  <Compile Remove="GplexBuffers.cs" />
+	  <PackageReference Include="System.Reflection.TypeExtensions" Version="4.7.0" />
+    <PackageReference Include="System.Runtime.Serialization.Formatters" Version="4.3.0" />
+  </ItemGroup>
+ 
+  <ItemGroup>
+    <PackageReference Include="YaccLexTools" Version="1.2.2" PrivateAssets="All" />

```

Comment:
In the `dotnet-tools.json` I install [YaccLexTools](https://www.nuget.org/packages/YaccLexTools). This tools has special item definitions for `YaccFiles` and `LexFiles`

The magic happens with the following definitions:
```
    <YaccFile Include="JmesPathParser.y">
      <OutputFile>JmesPathParser.g.cs</OutputFile>
    </YaccFile>
    <LexFile Include="JmesPathScanner.lex">
      <OutputFile>JmesPathScanner.g.cs</OutputFile>
    </LexFile>

```
This will run gppg on the YaccFile and gplex on the LexFile and generate the `.g.cs` files during the build.
This will also produce a GplexBuffers.cs file and a ShiftReduceParserCode file.

More information is [here](https://devnotes.ernstc.net/yacclextools)